frontend/src/app/shared/components/table/table.sass

```diff
@@ -8,7 +8,7 @@
     text-align: center
 
     &_heading
-      background-color: var(--fc-neutral-bg-color)
+      background-color: var(--display-gray-bgColor-muted)
```

Comment:
Why was this change necessary? the heading is now pretty dark

<img width="500" alt="Bildschirmfoto 2024-07-17 um 12 18 41" src="https://github.com/user-attachments/assets/e719b447-a99f-4957-9c1c-d40dd9e60e6b">

**Before**
<img width="500" alt="Bildschirmfoto 2024-07-17 um 12 18 27" src="https://github.com/user-attachments/assets/f0e9b253-0f4a-4583-84be-80879098d3a0">

Reply:
Hi @HDinger 
--fc-neutral-text-color is a fullcalendar variable, can we use it for a table?
I think that it was set by mistake.